js/main.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @description Return a random integer from min(included) to max(included)
```

Comment:
```suggestion
 * Return a random integer from min(included) to max(included)
```
description не обязательно писать, он и так поймет.